main/zapisy.py

```diff
@@ -152,3 +157,20 @@ def nauczyciel(id):
                            nazwisko=dane_nauczyciela['nazwisko'],
     )
 
+
+# Email confirmation
+@bp.route('/confirm_email/<given_confirmation_code>', methods=('GET', 'POST'))
```

Comment:
Tu nie powinno być POSTa, to nigdzue nie jest wykorzystywane.

Reply:
Gdyby ktoś z jakiegoś powodu chciał zrobić POSTa na ten URL, to też powinien móc.